### ISSUE TITLE
interesting but ultimately did nothing

### ISSUE THREAD

**Issue by danielb2** [svenanders/learn-react#10]
I followed the instructions, browsed to http://localhost:8080 and the page loaded, but nothing worked when I clicked it

**Comment by svenanders** [comment]
Yeah, I'm sorry. This repo is deprecated (as mentioned in the readme). It was a nice starting point a year and a half ago, but there's too much cruft here so I don't recommend using it.

**Comment by danielb2** [comment]
NB was meaningless for me as a header, so I didn't even read that part. Maybe make it more obvious?

**Comment by svenanders** [comment]
OK, will do!
By the way, this is a more interesting starting point: https://github.com/svenanders/minimal-react

It's the bare basics, with a dev+prod config. I've found that guidelines, recommended data flow libs etc change so fast that it's meaningless to include this stuff in a boilerplate.

**Comment by danielb2** [comment]
Cool, I will check that out. Thanks! :)
